debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/ehcache/EhcacheLogMinerEventProcessor.java

```diff
@@ -106,7 +106,22 @@ public LogMinerCache<String, String> getProcessedTransactionsCache() {
 
     @Override
     public void close() throws Exception {
+        transactionCache.close();
+
         if (cacheManager != null) {
```

Comment:
Not sure if there should be a try/finally around any of this.

Reply:
I can actually remove that call to `close` as it was used to synchronize Infinispan writes before shutdown, but since I reverted back to synchronous cache operations, I can remove this call.